canonical/ubuntu.com
# [copy update] /landscape/managed

## Done

- [Updated copy as per [doc](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit#)
- Drive bys:
  - Fixed section alignment in /landscape/managed to match [design](https://github.com/canonical/ubuntu.com/pull/12812#issuecomment-1523023623) 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12916.demos.haus/landscape/install
- See that requested changed have been made
- View the site locally in your web browser at: https://ubuntu-com-12916.demos.haus/landscape/managed
- See that the page matches design  

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3987
